diffulex_kernel/python/fused_moe_triton.py

```
@@ -0,0 +1,343 @@
+    # ── Data pointers ──
+    hidden_states_ptr,       # (M, H)
+    w13_ptr,                 # (E, 2*I, H)
+    intermediate_ptr,        # (num_tokens_post_pad, I)
+    # ── Routing index arrays ──
+    sorted_token_ids_ptr,    # (num_tokens_post_pad,)
+    expert_ids_ptr,          # (num_tiles_m,)
+    num_tokens_post_pad,     # scalar
+    # ── Dims ──
+    M,                       # number of real tokens (for bounds checking)
+    H: tl.constexpr,
+    I: tl.constexpr,
```

Ruff (0.15.7)

[error] Ambiguous variable name: `I`

(E741)